src/Workspaces/CoreTest/Renamer/RenamerTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.Renamer
+{
+    [UseExportProvider]
+    public partial class RenamerTests : TestBase
```

Comment:
```suggestion
    public abstract partial class RenamerTests : TestBase
```

Since I don't think there are any tests in here directly?